openbas-api/src/main/java/io/openbas/rest/exercise/ExerciseApi.java

```diff
@@ -380,20 +378,20 @@ public Exercise updateExerciseLessons(
 
   @DeleteMapping(EXERCISE_URI + "/{exerciseId}")
   @PreAuthorize("isExercisePlanner(#exerciseId)")
-  @Transactional(rollbackOn = Exception.class)
+  @Transactional(rollbackFor = Exception.class)
   public void deleteExercise(@PathVariable String exerciseId) {
     exerciseRepository.deleteById(exerciseId);
   }
 
   @GetMapping(EXERCISE_URI + "/{exerciseId}")
   @PreAuthorize("isExerciseObserver(#exerciseId)")
```

Comment:
Could we renme this isExerciseObserver function as well, maybe ? (or duplicate it like i did and mark the old one as deprecated?)

Reply:
Good point, I did it :)